react-components/src/architecture/base/renderTarget/CommandsController.ts

```diff
@@ -35,6 +36,10 @@ export class CommandsController extends PointerEvents {
   // ==================================================
 
   public get activeTool(): BaseTool | undefined {
+    return this._activeTool();
+  }
+
+  public get activeToolSignal(): Signal<BaseTool | undefined> {
     return this._activeTool;
   }
```

Comment:
This is so nice! I guess we don't technically need the `activeTool` then? Can't we just only use the active tool with the signal?